src/main/java/networkbook/logic/commands/delete/DeletePersonCommand.java

```diff
@@ -28,7 +28,8 @@ public class DeletePersonCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1\n"
             + "Usage 2: " + COMMAND_WORD + " [LIST INDEX OF CONTACT] [FIELD PREFIX]\n"
             + "Example: " + COMMAND_WORD + " 1 /priority\n"
-            + "If the field can have multiple values, /index must be used to specify index of the entry to delete.\n"
+            + "If the field can have multiple values, /index must be used to specify index of the entry to delete."
+            + "If /index is not specified, defaults to 1.\n"
```

Comment:
I'm not sure if this change is too much. Maybe can just leave the message as `If the field can have multiple values, /index can be used to specify index of the entry to delete.` I think would be safer this way, and the tester should be referring to the user guide anyway. We can add this as one of the planned enhancements in DG.